# 轴距
wheelbase: 1.054
# 轮距（前后轮相同）
tread: 1.62275
# 轮胎半径
wheel_radius: 0.1175
